config/environments/production.rb

```diff
@@ -83,4 +79,22 @@
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
+
+  # Sending e-mails is required for user management and registration e-mails
+  config.action_mailer.default_url_options = { host: config.wcrs_back_office_url, protocol: "http" }
+
+  # Don't care if the mailer can't send (if set to false)
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.delivery_method = :smtp
+
+  # Default settings are for mailcatcher
```

Comment:
In production can we remove the default settings? Should we forget to set the env vars the error is likely to point more to that, rather than because it failed to send using the credentials provided.

Reply:
Fixed!